appengine/examples.py

```diff
@@ -104,3 +99,74 @@ def get_example(name):
 examples = { 'datalog' : datalog_examples,
              'myrial' : myria_examples,
              'sql' : sql_examples }
+
+
+
+load_twitterk_data = '''T1 = load("s3://uwdb/sampleData/TwitterK.csv",
```

Comment:
I completely forgot about the S3 roles. Is that Hadoop switch difficult to do? Alternatively, if I changed these S3 URLs to http endpoints, would the lack of credentials still be a problem?